jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -70,6 +71,20 @@ public synchronized static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	/**
+	 * Queries if CRIU Security provider needs to be disabled
+	 * when CRIU support is enabled.
+	 *
+	 * @return true if needed, false otherwise
```

Comment:
This description is less clear than it could be; perhaps
```java
	/**
	 * Queries if CRIU Security provider needs to be disabled
	 * when CRIU support is enabled.
	 *
	 * @return true if disabled, otherwise false
	 */
```

Reply:
Updated

Reply:
That javadoc needs to be updated to match the new method name and behavior.

Reply:
Updated